android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMKeyboardDownloaderActivity.java

```diff
@@ -392,10 +462,21 @@ public static boolean isCustom(String u) {
     JSONArray fontSource = jsonFont.optJSONArray(KMManager.KMKey_FontSource);
     if (fontSource != null) {
       int fcCount = fontSource.length();
+      boolean containsTTF = hasTTFFont(fontSource);
       for (int i = 0; i < fcCount; i++) {
         String fontSourceString;
         try {
           fontSourceString = fontSource.getString(i);
+          // Prioritize .ttf if it exists
+          if (!FileUtils.isTTFFont(fontSourceString) && containsTTF) {
+            if (Build.VERSION.SDK_INT > 19) {
+              fontSource.remove(i);
```

Comment:
Won't this cause an index overrun with the `for` loop?

Reply:
This block in fontUrls was where I originally started the fix.
But now that it's handled earlier in ll 300-306 I should be able to revert this.